### DISCUSSION TITLE
Why does the exitstatus parameter in the pytest_terminal_summary hook function return the number 0 when the test passes and an enumerated value in other cases?

### DISCUSSION THREAD

**Discussion by backtracker** [pytest-dev/pytest#12088] [Q&A]
#: Tests passed.
    OK = 0
    #: Tests failed.
    TESTS_FAILED = 1
    #: pytest was interrupted.
    INTERRUPTED = 2
    #: An internal error got in the way.
    INTERNAL_ERROR = 3
    #: pytest was misused.
    USAGE_ERROR = 4
    #: pytest couldn't find tests.
    NO_TESTS_COLLECTED = 5

**Comment by RonnyPfannschmidt** [comment]
Programm return codes indicate errors in general

0 means no error
